Aspire Budgeting/ViewModels/FileSelectorViewModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+// FileSelectorViewModel.swift
+// Aspire Budgeting
+//
+
+import Foundation
+
+protocol FileSelectorViewModelDelegate: class {
+  func verify(file: File)
+}
+
+enum FileSelectorViewModelState {
+  case isLoading
+  case loaded
+  case error
+}
+
+struct FileSelectorViewModel {
+  let state: FileSelectorViewModelState
+  var files: [File]? = nil
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/02d5375236d0328ac5fc8dbb7c000406514d930c/Aspire Budgeting/ViewModels/FileSelectorViewModel.swift#L20"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>FileSelectorViewModel.swift:20</code></td>
    </tr>
  </tbody>
</table>